### ISSUE TITLE
Olena Penn w09 HW: Cover Letter and 2nd blog

### ISSUE THREAD

**Issue by op9674a** [ga-students/wdiroutcomes#129]
Cover Letter: https://drive.google.com/file/d/1psA30AXsUNhuW7MjDQojlY-JrbpKhQfu/view?usp=sharing

Job link: https://ikehawaii.efficientapply.com/jobs/207877.html

Blog: https://medium.com/@olenapenn/things-to-do-82d66352fcce

**Comment by nicoleump** [comment]
Thanks Olena! 

Great second post. Sleep is nice. :)

My feedback for your cover letter is as follows:
- First paragraph, great. 
- Second paragraph - you immediately put yourself into the newbie category by saying "I have just completed General Assembly’s Immersive Web Development program,". I recommend you reword it to say something like "For the last four months I have been leaving and breathing programming. I am focused on JavaScript but also delved into SQL, Ruby on Rails, and Angular, and I could not be happier that programming is my career path."
-Other than that, great job!
